tutorial.md

```diff
@@ -1,461 +0,0 @@
-# CRI-O Tutorial
```

Comment:
Are there links to this document on other sites that might go to 404 with this change?  Might be good to have "This page has moved, for cni instructions go here, for crio setup go here, etc.  Something short and sweet with appropriate links or link.

Reply:
Ok, after looking further, I would like to see this changed to a small Tutorials home page with links to the other new tutorial pages.